src/editor/editor.cpp

```diff
@@ -315,12 +286,23 @@ namespace Editor
             ImGui::Separator();
             ImGui::Text("Tile Set Properties");
 
-
-            if (ImGui::Button("Load Tileset"))
+            ImGui::InputText(" ", image_file_path, IM_ARRAYSIZE(image_file_path), ImGuiInputTextFlags_ReadOnly);
```

Comment:
From ImGui documentation for IM_ARRAYSIZE: Size of a static C-style array. Don't use on pointers!

Reply:
maybe use strlen()